test/integration/test_integration_basics.py

```diff
@@ -192,11 +192,11 @@ def test_https_h1_tls_context_configuration(https_test_server_fixture):
 
   parsed_json, _ = https_test_server_fixture.runNighthawkClient([
       "--h2", "--duration 1",
-      "--tls-context {common_tls_context:{tls_params:{cipher_suites:[\"-ALL:ECDHE-RSA-AES256-GCM-SHA384\"]}}}",
+      "--tls-context {common_tls_context:{tls_params:{cipher_suites:[\"-ALL:ECDHE-RSA-CHACHA20-POLY1305\"]}}}",
       https_test_server_fixture.getTestServerRootUri()
   ])
   counters = https_test_server_fixture.getNighthawkCounterMapFromJson(parsed_json)
-  assertEqual(counters["ssl.ciphers.ECDHE-RSA-AES256-GCM-SHA384"], 1)
+  assertEqual(counters["ssl.ciphers.ECDHE-RSA-CHACHA20-POLY1305"], 1)
```

Comment:
Arguably changing ciphers will create differences in performance. I think for now this is fine, but please do take on the long term fix.

Reply:
Added https://github.com/envoyproxy/nighthawk/issues/156 for tracking